core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/dot-edit-ema-workflow-actions/dot-edit-ema-workflow-actions.component.html

```diff
@@ -0,0 +1,8 @@
+@if (actions()) {
+<dot-workflow-actions
+    [loading]="loading()"
+    [actions]="actions()"
+    (actionFired)="handleActionTrigger($event)" />
+} @else {
+<p-button [loading]="true" [label]="'Loading' | dm" data-testId="loading-button" />
+}
```

Comment:
```suggestion
@if (actions()) {
  <dot-workflow-actions
      [loading]="loading()"
      [actions]="actions()"
      (actionFired)="handleActionTrigger($event)" />
  } @else {
    <p-button [loading]="true" [label]="'Loading' | dm" data-testId="loading-button" />
  }
```

Reply:
@fmontes I can add the space in the `@if/@else`, the pre-commit removes them.

Reply:
WHAT!!!! @oidacra we need to do something here 🤣